README.md

```diff
@@ -119,6 +117,20 @@ Create docker image:
   docker-compose build
 ```
 
+### Running Locally (Recommended)
+
+For this approach, the database must still be served via docker:
+```bash
+  docker-compose up snl-db
```

Comment:
you can still do just `docker-compose up` and a separate gradle run, but probably just running db will be faster